### PR TITLE
Revert profile pictures

### PR DESCRIPTION
**Pull request by dav1312**
Revert profile pictures of people without custom profile pictures

### CODE REVIEW

File: src/main.css [hunk 1]
```diff
@@ -211,4 +211,11 @@ body {
 /* Green radio bars (privacy and safety settings) */
 .radioBar-bMNUI-[style="--radio-bar-accent-color:#3ba55c; padding: 10px;"] {
   --radio-bar-accent-color: var(--old-green) !important;
-}
\ No newline at end of file
+}
+
+/* Profile pictures */
+[src="/assets/1f0bfc0865d324c2587920a7d80c609b.png"] {content: url(/assets/6debd47ed13483642cf09e832ed0bc1b.png)}
```
Comment (CanadaHonk) [inline src/main.css]: Please use similar spacing and syntax with the rest of the file.

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

**Review by CanadaHonk — APPROVED** [review]

**Comment by CanadaHonk** [comment]
Sorry to ask further, could you use the new restructure. Please add the changes to a file like `src/components/pfps.css`.

**Comment by DorCoMaNdO** [comment]
Might I request a similar fix for group chats? The "vibrant" colors are a lot more eye catching when I glance at Discord and it tends to be annoying.

**Comment by CanadaHonk** [comment]
> Might I request a similar fix for group chats? The "vibrant" colors are a lot more eye catching when I glance at Discord and it tends to be annoying.

What pictures? Please make a new issue with examples, thanks.

**Comment by CanadaHonk** [comment]
> Sorry to ask further, could you use the new restructure. Please add the changes to a file like `src/components/pfps.css`.

Doing this myself; but still thanks for the PR!

**Pull request closed without merging**